### PR TITLE
Use a single column layout and don't load primary images for webinar pages.

### PR DESCRIPTION
**Pull request by Shinsina**
webinar-layout-1
webinar-layout-2
webinar-layout-3

### CODE REVIEW

**Comment by Shinsina** [comment]
[Change to Webinar Pages](https://trello.com/c/QWqBTWOm/408-change-to-webinar-pages)

File: packages/global/components/layouts/content/webinar.marko [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+                aliases=aliases
+                block-name=blockName
+                content=content
+              />
+            </if>
+
+            $ const sidebars = getAsArray(content, "sidebars").map((sidebar) => sidebar.body);
+            <marko-web-content-sidebars block-name=blockName obj={ sidebars } />
+
+            <if(leadersContentTypes.includes(content.type))>
```
Comment (B77Mills) [inline packages/global/components/layouts/content/webinar.marko:98]: This if check can be removed along with the one on line 144.  Please also remove the const on line 9.

File: packages/global/components/layouts/content/webinar.marko [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+    <theme-gam-define-display-ad
+      name="leaderboard"
+      position="content_page"
+      aliases=aliases
+      modifiers=["inter-block"]
+    />
+    <theme-gam-wallpaper-ad aliases=aliases position="content_page" />
+  </@section>
+
+  <@section|{ blockName, content, aliases, authors }| modifiers=["content-header"]>
```
Comment (B77Mills) [inline packages/global/components/layouts/content/webinar.marko:31]: authors can be removed here along with lines 36 - 46.  I webinars do not have an author field.

File: packages/global/components/layouts/content/webinar.marko [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+      <global-sponsored-section-logo alias=primarySection.alias modifiers=["content-page"] class="mt-3" />
+        <marko-web-social-sharing
+          path=content.siteContext.path
+          providers=["print", "facebook", "linkedin", "twitter", "pinterest"]
+          print-path=`/print/content/${content.id}`
+        />
+    </div>
+  </@section>
+
+  <@section|{ aliases, content, authors }| modifiers=["content-body"]>
```
Comment (B77Mills) [inline packages/global/components/layouts/content/webinar.marko:56]: remove authors from here and line 102 - 106

**Review by B77Mills — CHANGES_REQUESTED** [review]

**Review by B77Mills — APPROVED** [review]

**Pull request merged**